Packs/SentinelOne/Scripts/SentinelOneGetMAC/SentinelOneGetMAC.py

```diff
@@ -0,0 +1,35 @@
+import demistomock as demisto  # noqa: F401
```

Comment:
Question - where would we use this script(?) because there is no playbook which is consuming this, is the script  intended to run from playground? If yes then would it make sense to have another integration command for this? Thanks!

Reply:
New command got added to the integration that replaces the script.